src/js/off-canvas.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+/*global document window*/
+
+(function() {
+
+  const html = document.querySelector('html');
+  const header = document.querySelector('.site-header');
+  const menu = document.querySelector('.site-header__menu');
+  const menu_open = document.querySelector('.site-header__menu-open');
+  const menu_close = document.querySelector('.site-header__menu-close');
```

Comment:
Thought: Wonder if we can devise some semantic selector approach for grabbing DOM elements, which might be more tied to their role (or decoupled from their classnames, e.g. `.js-site-header__menu-open` or `[data-site-header-menu-open]`). What do you think?

Reply:
Agreed.

Using data attributes now.